DATA/production/configurations/asyncReco/async_pass.sh

```diff
@@ -340,31 +340,40 @@ if [[ -n "$ALIEN_JDL_USEGPUS" && $ALIEN_JDL_USEGPUS != 0 ]]; then
   export GPUTYPE="HIP"
   export GPUMEMSIZE=$((25 << 30))
   if [[ "0$ASYNC_PASS_NO_OPTIMIZED_DEFAULTS" != "01" ]]; then
-    if [[ $keep -eq 0 ]]; then
+    if [[ "ALIEN_JDL_USEFULLNUMADOMAIN" == 0 ]]; then
+      if [[ $keep -eq 0 ]]; then
+	if [[ $ALIEN_JDL_UNOPTIMIZEDGPUSETTINGS != 1 ]]; then
+	  export OPTIMIZED_PARALLEL_ASYNC=pp_1gpu  # sets the multiplicities to optimized defaults for this configuration (1 job with 1 gpu on EPNs)
+	else
+	  # forcing multiplicities to be 1
+	  export MULTIPLICITY_PROCESS_tof_matcher=1
+	  export MULTIPLICITY_PROCESS_mch_cluster_finder=1
+	  export MULTIPLICITY_PROCESS_tpc_entropy_decoder=1
+	  export MULTIPLICITY_PROCESS_itstpc_track_matcher=1
+	  export MULTIPLICITY_PROCESS_its_tracker=1
+	fi
+	export TIMEFRAME_RATE_LIMIT=8
+      else
+	export TIMEFRAME_RATE_LIMIT=4
+      fi
       if [[ $ALIEN_JDL_UNOPTIMIZEDGPUSETTINGS != 1 ]]; then
-	export MULTIPLICITY_PROCESS_tof_matcher=2
-	export MULTIPLICITY_PROCESS_mch_cluster_finder=3
-	export MULTIPLICITY_PROCESS_tpc_entropy_decoder=2
-	export MULTIPLICITY_PROCESS_itstpc_track_matcher=3
-	export MULTIPLICITY_PROCESS_its_tracker=2
+	export OMP_NUM_THREADS=8
```

Comment:
The OPTIMIZED_PARALLEL_ASYNC also sets the num threads (no longer globally, but per process).
Thus you can remove the OMP_NUM_THREADS here. And the OMP_NUM_THREADS=4 should go above to the "# forcing multiplicities to be 1"

Reply:
done